src/Game/AccountManagement.java

```diff
@@ -271,20 +530,422 @@ public static void panel_welcome_hook(int n) {
+	        Panel.setFocus(Client.panelContactDetails,Client.fullNameInput);
+	  } catch (Exception e) {
+		  e.printStackTrace();
+	  }
+  }
+
+  public static int options_security_hook(int xPos, int yPos, int mouseX, int mouseY) {
+    short uiWidth = 196;
+    int currYPos = yPos;
+    if (Settings.SHOW_ACCOUNT_SECURITY_SETTINGS.get(Settings.currentProfile)) {
+      currYPos += 5;
+      Renderer.drawString("Security settings", xPos, currYPos, 1, 0);
+      currYPos += 15;
+      int textColor = 16777215;
+      if (mouseX > xPos
+          && mouseX < xPos + uiWidth
+          && mouseY > currYPos - 12
+          && mouseY < currYPos + 4) {
+        textColor = 16776960;
+      }
+      Renderer.drawString("Change password", xPos, currYPos, 1, textColor);
+      currYPos += 15;
+      textColor = 16777215;
+      if (mouseX > xPos
+          && mouseX < xPos + uiWidth
+          && mouseY > currYPos - 12
+          && mouseY < currYPos + 4) {
+        textColor = 16776960;
+      }
+      Renderer.drawString("Change recovery questions", xPos, currYPos, 1, textColor);
+      currYPos += 15;
+      textColor = 16777215;
+      if (mouseX > xPos
+          && mouseX < xPos + uiWidth
+          && mouseY > currYPos - 12
+          && mouseY < currYPos + 4) {
+        textColor = 16776960;
+      }
+      Renderer.drawString("Change contact details", xPos, currYPos, 1, textColor);
+      currYPos += 15;
+      // currYPos += 5; //originally had this extra space but makes moves privacy settings and
+      // subsequent block
+      // a bit down with respect to when SHOW_ACCOUNT_SECURITY_SETTINGS is not enabled
+    }
+    return currYPos - yPos;
+  }
+
+  public static int options_security_click_hook(
+      int xPos, int yPos, int mouseX, int mouseY, int mouseButtonClick) {
+    short uiWidth = 196;
+    int currYPos = yPos;
+    if (Settings.SHOW_ACCOUNT_SECURITY_SETTINGS.get(Settings.currentProfile)) {
+      currYPos += 15;
+      currYPos += 20;
+      if (mouseX > xPos
+          && mouseX < xPos + uiWidth
+          && mouseY > currYPos - 12
+          && mouseY < currYPos + 4
+          && mouseButtonClick == 1) {
+        AccountManagement.panelPasswordChangeMode = 6;
+        Client.modal_enteredText = "";
+        Client.modal_text = "";
+      }
+      currYPos += 15;
+      if (mouseX > xPos
+          && mouseX < xPos + uiWidth
+          && mouseY > currYPos - 12
+          && mouseY < currYPos + 4
+          && mouseButtonClick == 1) {
+        StreamUtil.newPacket(197);
+        StreamUtil.sendPacket();
+      }
+      currYPos += 15;
+      if (mouseX > xPos
+          && mouseX < xPos + uiWidth
+          && mouseY > currYPos - 12
+          && mouseY < currYPos + 4
+          && mouseButtonClick == 1) {
+        StreamUtil.newPacket(247);
+        StreamUtil.sendPacket();
+      }
+      currYPos += 15;
+      currYPos +=
+          30; // originally was += 35 but with the skip tutorial added, clicking logout doesn't do

```

Comment:
will need to be += 35 if we fix spacing above

Reply:
will need to maybe hook and adjust the skip tutorial button bounds. :-\

Reply:
fixed